install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
One of the reasons we explicitly used bash was because we couldn't get the colour codes working correctly in Dash, which Ubuntu and Debian use by default. I tried tput a while back, but that caused other issues. Could you please get the colour codes working too?